src/actions/collectiblesActions.js

```diff
@@ -181,9 +189,43 @@ export const fetchCollectiblesHistoryAction = () => {
   };
 };
 
-export const fetchAllCollectiblesDataAction = () => {
+
+export const fetchAllAccountsCollectiblesAction = () => {
+  return async (dispatch: Dispatch, getState: GetState) => {
+    const {
+      accounts: { data: accounts },
+    } = getState();
+
+    const promises = accounts.map(async account => {
+      await dispatch(fetchCollectiblesAction(account));
+    });
+    await Promise.all(promises).catch(_ => _);
```

Comment:
What do you think about:
```suggestion
    await Promise.allSettled(promises).catch(_ => _);
```